thinc/backends/__init__.py

```diff
@@ -91,10 +95,15 @@ def get_ops(name: str, **kwargs) -> Ops:
     cls: Optional[Callable[..., Ops]] = None
     if name == "cpu":
         _import_extra_cpu_backends()
+        
         cls = ops_by_name.get("apple", ops_by_name.get("numpy"))
+        
+        if "bigendian" in ops_by_name:
+            cls = ops_by_name.get("bigendian", ops_by_name.get("numpy"))
+
```

Comment:
```suggestion
        cls = ops_by_name.get("numpy")
        cls = ops_by_name.get("apple", cls)
        cls = ops_by_name.get("bigendian", cls)
```

Reply:
This is where a ranking from the ops classes themselves would be better.

Reply:
Updated with suggested change and retested.